integration/airflow/openlineage/airflow/adapter.py

```diff
@@ -44,8 +44,8 @@
 # https://openlineage.io/docs/spec/facets/job-facets/job-type
 # They must be set after the `set_producer(_PRODUCER)`
 # otherwise the `JobTypeJobFacet._producer` will be set with the default value
-_JOB_TYPE_DAG = JobTypeJobFacet(jobType="DAG", integration="AIRFLOW", processingType="BATCH")
-_JOB_TYPE_TASK = JobTypeJobFacet(jobType="TASK", integration="AIRFLOW", processingType="BATCH")
+_JOB_TYPE_DAG = job_type_job.JobTypeJobFacet(jobType="DAG", integration="AIRFLOW", processingType="BATCH")
+_JOB_TYPE_TASK = job_type_job.JobTypeJobFacet(jobType="TASK", integration="AIRFLOW", processingType="BATCH")
```

Comment:
nit: we could possibly just pass the producer as an argument now, it should be simpler.

Reply:
nit: the _PRODUCER on line 39 is two strings now, i think we can clean that

Reply:
> nit: we could possibly just pass the producer as an argument now, it should be simpler.

I don't think we're going away with `set_producer` approach? For me it's still main way to set producer, maybe I'm wrong 🤔 

> nit: the _PRODUCER on line 39 is two strings now, i think we can clean that
It's not two string but the concatenation might look better, agreed.

Reply:
> It's not two string but the concatenation might look better, agreed.

That's exactly what i meant :)

> I don't think we're going away with set_producer approach? For me it's still main way to set producer, maybe I'm wrong 🤔

We are not, I don't liked the idea that we have to rely on defining something (facets) after calling something (set_producer) if there is another, more reliable way of making sure it works as expected. Just a preference, so no pressure here.